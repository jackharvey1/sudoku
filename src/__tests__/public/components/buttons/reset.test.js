import React from 'react';
import ResetButton from '../../../../public/components/buttons/reset';
import Renderer from 'react-test-renderer';

describe('Reset button', () => {
    it('renders as intended', () => {
        const testRenderer = Renderer.create(
            <ResetButton onClick={() => null} />
        );
        expect(testRenderer.toJSON()).toMatchSnapshot();
    });
});
