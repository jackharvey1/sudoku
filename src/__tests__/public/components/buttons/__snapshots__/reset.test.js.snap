// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reset button renders as intended 1`] = `
<button
  className="css-mhex0s"
  onClick={[Function]}
>
  <div
    className="css-1sginf6"
  >
    <svg
      className="css-1w0ds7r"
      fill="currentColor"
      height="1em"
      preserveAspectRatio="xMidYMid meet"
      style={
        Object {
          "color": undefined,
          "verticalAlign": "middle",
        }
      }
      viewBox="0 0 40 40"
      width="1em"
    >
      <g>
        <path
          d="m27.5 27.5l-8.1-8.7h5.8c-0.6-5.1-5-8.8-10.2-8.8-5.5 0-10 4.5-10 10s4.5 10 10 10c2.2 0 4.1-0.6 5.9-1.9l0.4-0.4 3.4 3.6-0.6 0.5c-2.6 2-5.7 3.2-9.1 3.2-7.1 0-13.1-5-14.6-11.6 0-0.1-0.1-0.2-0.1-0.3v-0.2s-0.1-0.2-0.1-0.2v-0.1c0-0.1 0-0.3 0-0.3-0.1-0.5-0.2-1-0.2-1.5v-1.6c0-0.4 0.1-1 0.2-1.5 0 0 0-0.2 0-0.3v-0.1c0 0 0.1 0 0.1-0.1v-0.3c0-0.1 0.1-0.2 0.1-0.3 1.6-6.6 7.5-11.6 14.6-11.6 0.9 0 1.6 0.1 2.4 0.2h0.3c4.5 0.8 8.3 3.7 10.4 7.5 1 1.8 1.7 3.9 1.8 6.1h5.1z"
        />
      </g>
    </svg>
    <p>
      Reset
    </p>
  </div>
</button>
`;
